tags.go

```diff
@@ -78,7 +78,7 @@ func Parse(tag string) (*Tags, error) {
 			return nil, errTagValueSyntax
 		}
 
-		key := string(tag[:i])
+		key := tag[:i]
```

Comment:
Could you remove these changes? I want to make sure only the doc comment changes are merged.

Reply:
@fatih 

Of course, I made as was as before changed.

Reply:
@fatih 

But these castings are truly redundant. Or I misunderstood? Can you explain please?

Reply:
I'm not against it, I just want to keep the changes separate :) Happy to merge them in another PR.

Reply:
@fatih Oh, thanks! 
I am making new PR already :)